mappers/BaselineMapper.rb

```diff
@@ -72,28 +72,113 @@ def create_osw(scenario, features, feature_names)
         end
         feature = features[0]
         feature_id = feature.id
-        feature_type = feature.feature_type
-        
+        feature_type = feature.type 
+
         feature_name = feature.name
         if feature_names.size == 1
           feature_name = feature_names[0]
         end
         
+        if feature_type == 'Building'
```

Comment:
@tanushree04 @rawadelkontar we should throw an error or something in this mapper if the feature type is not building